Package.swift

```diff
@@ -28,7 +28,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/Kitura/BlueRSA.git", from: "1.0.31"),
+        .package(url: "https://github.com/Kitura/BlueRSA.git", from: "1.0.200"),
         .package(url: "https://github.com/Kitura/BlueCryptor.git", from: "1.0.0"),
```

Comment:
https://github.com/Kitura/CommonCrypto.git needs to be updated (new tag) before merging this PR, as it seems the package manager for Swift 4 is confused by the old origin (IBM-Swift).
Then the tag probably need to be propagated to BlueCryptor and updated here also.